core/domain/topic_services.py

```diff
@@ -386,6 +395,24 @@ def apply_change_list(topic_id, change_list):
         raise
 
 
+def check_subtopic_url_fragment_uniqueness(url_fragment, topic):
```

Comment:
See above re framing of "uniqueness".

Should this be a method on the topic object instead? I.e. topic.does_subtopic_exist() or similar